py3status/modules/getjson.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Query a url and display the json response
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds
+        (default 30)
+    format: placeholders will be replaced by the returned json key names
+        (default '')
+    negative_cache_timeout: how often to check again when offline
+        (default 2)
+    params: specify the parameters to pass to the url as a dict
+        (default {})
+    sep: the separator between parent and child objects
+        (default '_')
+    timeout: how long before deciding we're offline
+        (default 5)
+    url: specify a url to fetch json from
+        (default '')
+
+Format placeholders:
+    Placeholders will be replaced by the json keys
+    Placeholders for objects with sub-objects are flattened using 'sep' in between
+    (eg. {'parent': {'child': 'value'}} will use placeholder {parent_child}
+
+@author vicyap
+"""
+
+import collections
+import json
+try:
+    # python3
+    from urllib.parse import urlencode
+    from urllib.request import urlopen
+except:
+    from urllib import urlencode
+    from urllib2 import urlopen
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 30
+    format = ''
+    negative_cache_timeout = 2
+    params = {}
+    sep = '_'
+    timeout = 5
+    url = ''
+
+    def _flatten(self, d, parent_key=None):
+        '''
+        modified from http://stackoverflow.com/a/6027615
+        '''
+        items = []
+        for k, v in d.iteritems():
+            new_key = parent_key + self.sep + k if parent_key else k
+            if isinstance(v, collections.Mapping):
+                items.extend(self._flatten(v, new_key).iteritems())
+            else:
+                items.append((new_key, v))
+        return dict(items)
```

Comment:
nice, however I could not think of a good way to handle lists. Currently list values are printed as `['item0', 'item1', ...]`

Do you have any suggestions on how to handle lists? Or whether or not this module needs to handle lists?